src/frontend/apps/drive/src/hooks/useThemeCustomization.tsx

```diff
@@ -3,14 +3,23 @@ import { ThemeCustomization } from "@/features/drivers/types";
 import { splitLocaleCode } from "@/features/i18n/utils";
 import { useTranslation } from "react-i18next";
 
-export const useThemeCustomization = (key: keyof ThemeCustomization) => {
+export const useThemeCustomizationFooter = () => {
   const { config } = useConfig();
   const { i18n } = useTranslation();
   const language = splitLocaleCode(i18n.language).language;
-  const themeCustomization = config?.theme_customization?.[key];
+  const themeCustomization = config?.theme_customization?.footer;
   return {
     ...themeCustomization?.default,
     ...(themeCustomization?.[language as keyof typeof themeCustomization] ??
       {}),
   };
 };
+
+export const useThemeCustomizationElementsDisplay = (key: keyof Pick<ThemeCustomization, 'auth_buttons' | 'language_picker'>) => {
+  const { config } = useConfig();
+  const themeCustomization = config?.theme_customization?.[key];
+
+  return {
+    ...(themeCustomization ?? {})
+  }
+}
```

Comment:
No need for this, too specific, we want to keep a generic way of fetching the conf